### ISSUE TITLE
Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false

### ISSUE THREAD

**Issue by Grothesk242** [iobroker-community-adapters/ioBroker.tado#128]
**Describe the bug**  
`Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false`

**To Reproduce**  
Log file keeps being flooded with this message: 
`Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false
Log entry is triggered for each thermostat. 

`

**Versions:**  
 - Adapter version: tado           - v0.2.4
 - JS-Controller version: 3.1.6
 - Node version: v15.4.0
 - Operating system: Raspberry OS Buster / 10

**Additional context**  
Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false

**Comment by Schmetterfliege** [comment]
Hi,

same issue for me. Below the logs from when I see errors (cannot connect to tado cloud and multiple times clearing the zoneOverlay for all rooms?!)
Manually restarting the adapter doesn't help.
Connects to the cloud successfully, but then it spams the logs again.
Makes me thing TADO servers are causing this...
Any idea why it is clearing all rooms at once sometimes? out of a sudden?

020-12-15 01:26:48.783 - info: tado.0 (5657) state tado.0.580325.Rooms.1.overlay.clearZoneOverlay deleted
--
2020-12-15 01:26:48.860 - info: tado.0 (5657) state tado.0.580325.Rooms.1.overlay.termination.typeSkillBasedApp deleted
2020-12-15 01:26:50.062 - info: tado.0 (5657) state tado.0.580325.Rooms.2.setting.power deleted
2020-12-15 01:26:50.074 - info: tado.0 (5657) state tado.0.580325.Rooms.2.overlay.clearZoneOverlay deleted
2020-12-15 01:26:50.171 - info: tado.0 (5657) state tado.0.580325.Rooms.2.overlay.termination.typeSkillBasedApp deleted
2020-12-15 01:26:51.380 - info: tado.0 (5657) state tado.0.580325.Rooms.3.setting.power deleted
2020-12-15 01:26:51.408 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.clearZoneOverlay deleted
2020-12-15 01:26:51.415 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.termination.typeSkillBasedApp deleted
2020-12-15 01:26:51.417 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.termination.durationInSeconds deleted
2020-12-15 01:26:52.539 - info: tado.0 (5657) state tado.0.580325.Rooms.4.setting.power deleted
2020-12-15 01:26:52.568 - info: tado.0 (5657) state tado.0.580325.Rooms.4.overlay.clearZoneOverlay deleted
2020-12-15 01:26:52.597 - info: tado.0 (5657) state tado.0.580325.Rooms.4.overlay.termination.typeSkillBasedApp deleted
2020-12-15 01:26:53.709 - info: tado.0 (5657) state tado.0.580325.Rooms.5.setting.power deleted
2020-12-15 01:26:53.738 - info: tado.0 (5657) state tado.0.580325.Rooms.5.overlay.clearZoneOverlay deleted
2020-12-15 01:26:53.745 - info: tado.0 (5657) state tado.0.580325.Rooms.5.overlay.termination.typeSkillBasedApp deleted
2020-12-15 01:26:54.886 - info: tado.0 (5657) state tado.0.580325.Rooms.6.setting.power deleted
2020-12-15 01:26:54.963 - info: tado.0 (5657) state tado.0.580325.Rooms.6.overlay.clearZoneOverlay deleted
2020-12-15 01:26:54.967 - info: tado.0 (5657) state tado.0.580325.Rooms.6.overlay.termination.typeSkillBasedApp deleted
2020-12-15 02:00:00.039 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 03:00:00.016 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 04:00:00.036 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 05:00:00.038 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 06:00:00.028 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 07:00:00.016 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 08:00:00.034 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 09:00:00.023 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 09:18:46.069 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Balkon Wohnbereich geöffnet
2020-12-15 09:18:46.071 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer = manual
2020-12-15 09:18:46.072 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer = manual
2020-12-15 09:18:46.076 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer Heizung ausgeschaltet
2020-12-15 09:18:46.078 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer Heizung ausgeschaltet
2020-12-15 09:18:46.096 - info: tado.0 (5657) Power changed for room : 1 in home : 580325 to API with : off and Temperature : 19 and mode : MANUAL
2020-12-15 09:18:46.115 - info: tado.0 (5657) Power changed for room : 2 in home : 580325 to API with : off and Temperature : 18 and mode : MANUAL
2020-12-15 09:29:12.268 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Balkon Wohnbereich geschlossen
2020-12-15 09:29:12.269 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer Heizung auf manual
2020-12-15 09:29:12.270 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer Heizung auf manual
2020-12-15 09:29:12.338 - info: tado.0 (5657) Power changed for room : 1 in home : 580325 to API with : on and Temperature : 19 and mode : MANUAL
2020-12-15 09:29:12.341 - info: tado.0 (5657) Power changed for room : 2 in home : 580325 to API with : on and Temperature : 18 and mode : MANUAL
2020-12-15 10:00:00.014 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 10:19:13.672 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Balkon Wohnbereich geöffnet
2020-12-15 10:19:13.673 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer = manual
2020-12-15 10:19:13.674 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer = manual
2020-12-15 10:19:13.676 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer Heizung ausgeschaltet
2020-12-15 10:19:13.676 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer Heizung ausgeschaltet
2020-12-15 10:19:13.685 - info: tado.0 (5657) Power changed for room : 1 in home : 580325 to API with : off and Temperature : 19 and mode : MANUAL
2020-12-15 10:19:13.735 - info: tado.0 (5657) Power changed for room : 2 in home : 580325 to API with : off and Temperature : 18 and mode : MANUAL
2020-12-15 10:25:36.781 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Balkon Wohnbereich geschlossen
2020-12-15 10:25:36.783 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer Heizung auf manual
2020-12-15 10:25:36.783 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer Heizung auf manual
2020-12-15 10:25:36.791 - info: tado.0 (5657) Power changed for room : 1 in home : 580325 to API with : on and Temperature : 19 and mode : MANUAL
2020-12-15 10:25:36.795 - info: tado.0 (5657) Power changed for room : 2 in home : 580325 to API with : on and Temperature : 18 and mode : MANUAL
2020-12-15 10:37:23.387 - error: tado.0 (5657) Error in data refresh : Error: Request failed with status code 500
2020-12-15 10:37:23.388 - error: tado.0 (5657) Disconnected from Tado cloud service ..., retry in 30 seconds !
2020-12-15 10:37:26.783 - info: tado.0 (5657) state tado.0.580325.Rooms.1.setting.power deleted
2020-12-15 10:37:26.795 - info: tado.0 (5657) state tado.0.580325.Rooms.1.overlay.clearZoneOverlay deleted
2020-12-15 10:37:26.857 - info: tado.0 (5657) state tado.0.580325.Rooms.1.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:37:28.212 - info: tado.0 (5657) state tado.0.580325.Rooms.2.setting.power deleted
2020-12-15 10:37:28.224 - info: tado.0 (5657) state tado.0.580325.Rooms.2.overlay.clearZoneOverlay deleted
2020-12-15 10:37:28.288 - info: tado.0 (5657) state tado.0.580325.Rooms.2.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:37:29.883 - info: tado.0 (5657) state tado.0.580325.Rooms.3.setting.power deleted
2020-12-15 10:37:29.959 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.clearZoneOverlay deleted
2020-12-15 10:37:29.969 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:37:29.972 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.termination.durationInSeconds deleted
2020-12-15 10:37:31.446 - info: tado.0 (5657) state tado.0.580325.Rooms.4.setting.power deleted
2020-12-15 10:37:31.524 - info: tado.0 (5657) state tado.0.580325.Rooms.4.overlay.clearZoneOverlay deleted
2020-12-15 10:37:31.533 - info: tado.0 (5657) state tado.0.580325.Rooms.4.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:37:32.794 - info: tado.0 (5657) state tado.0.580325.Rooms.5.setting.power deleted
2020-12-15 10:37:32.866 - info: tado.0 (5657) state tado.0.580325.Rooms.5.overlay.clearZoneOverlay deleted
2020-12-15 10:37:32.876 - info: tado.0 (5657) state tado.0.580325.Rooms.5.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:37:34.241 - info: tado.0 (5657) state tado.0.580325.Rooms.6.setting.power deleted
2020-12-15 10:37:34.269 - info: tado.0 (5657) state tado.0.580325.Rooms.6.overlay.clearZoneOverlay deleted
2020-12-15 10:37:34.280 - info: tado.0 (5657) state tado.0.580325.Rooms.6.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:37:53.903 - info: tado.0 (5657) Connected to Tado cloud, initialyzing ...
2020-12-15 10:38:54.449 - error: tado.0 (5657) Error in data refresh : Error: Request failed with status code 504
2020-12-15 10:38:54.450 - error: tado.0 (5657) Disconnected from Tado cloud service ..., retry in 30 seconds !
2020-12-15 10:39:25.053 - info: tado.0 (5657) Connected to Tado cloud, initialyzing ...
2020-12-15 10:40:25.555 - error: tado.0 (5657) Error in data refresh : Error: Request failed with status code 504
2020-12-15 10:40:25.556 - error: tado.0 (5657) Disconnected from Tado cloud service ..., retry in 30 seconds !
2020-12-15 10:40:30.795 - info: javascript.0 (5717) script.js.Thermostate.Schlafzimmer_Heizung: KEVQU: Schlafzimmer Fenster geöffnet
2020-12-15 10:40:30.796 - info: javascript.0 (5717) script.js.Thermostate.Schlafzimmer_Heizung: KEVQU: Schlafzimmer Heizung ausgeschaltet
2020-12-15 10:40:40.417 - info: javascript.0 (5717) script.js.Thermostate.Schlafzimmer_Heizung: KEVQU: Schlafzimmer Fenster geschlossen
2020-12-15 10:40:40.419 - info: javascript.0 (5717) script.js.Thermostate.Schlafzimmer_Heizung: KEVQU: Schlafzimmer Heizung auf auto
2020-12-15 10:40:40.434 - error: tado.0 (5657) Issue at state change : TypeError: Cannot read property 'val' of null
2020-12-15 10:40:56.072 - info: tado.0 (5657) Connected to Tado cloud, initialyzing ...
2020-12-15 10:41:56.561 - error: tado.0 (5657) Error in data refresh : Error: Request failed with status code 504
2020-12-15 10:41:56.562 - error: tado.0 (5657) Disconnected from Tado cloud service ..., retry in 30 seconds !
2020-12-15 10:42:27.116 - info: tado.0 (5657) Connected to Tado cloud, initialyzing ...
2020-12-15 10:43:27.574 - error: tado.0 (5657) Error in data refresh : Error: Request failed with status code 504
2020-12-15 10:43:27.575 - error: tado.0 (5657) Disconnected from Tado cloud service ..., retry in 30 seconds !
2020-12-15 10:43:58.097 - info: tado.0 (5657) Connected to Tado cloud, initialyzing ...
2020-12-15 10:44:33.803 - info: tado.0 (5657) Initialisation finished, connected to Tado Cloud service refreshing every : 10 seconds
2020-12-15 10:55:20.505 - info: tado.0 (5657) state tado.0.580325.Rooms.1.setting.power deleted
2020-12-15 10:55:20.518 - info: tado.0 (5657) state tado.0.580325.Rooms.1.overlay.clearZoneOverlay deleted
2020-12-15 10:55:20.537 - info: tado.0 (5657) state tado.0.580325.Rooms.1.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:55:22.041 - info: tado.0 (5657) state tado.0.580325.Rooms.2.setting.power deleted
2020-12-15 10:55:22.075 - info: tado.0 (5657) state tado.0.580325.Rooms.2.overlay.clearZoneOverlay deleted
2020-12-15 10:55:22.121 - info: tado.0 (5657) state tado.0.580325.Rooms.2.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:55:23.617 - info: tado.0 (5657) state tado.0.580325.Rooms.3.setting.power deleted
2020-12-15 10:55:23.670 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.clearZoneOverlay deleted
2020-12-15 10:55:23.681 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:55:23.684 - info: tado.0 (5657) state tado.0.580325.Rooms.3.overlay.termination.durationInSeconds deleted
2020-12-15 10:55:25.006 - info: tado.0 (5657) state tado.0.580325.Rooms.4.setting.power deleted
2020-12-15 10:55:25.077 - info: tado.0 (5657) state tado.0.580325.Rooms.4.overlay.clearZoneOverlay deleted
2020-12-15 10:55:25.087 - info: tado.0 (5657) state tado.0.580325.Rooms.4.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:55:26.413 - info: tado.0 (5657) state tado.0.580325.Rooms.5.setting.power deleted
2020-12-15 10:55:26.447 - info: tado.0 (5657) state tado.0.580325.Rooms.5.overlay.clearZoneOverlay deleted
2020-12-15 10:55:26.452 - info: tado.0 (5657) state tado.0.580325.Rooms.5.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:55:27.784 - info: tado.0 (5657) state tado.0.580325.Rooms.6.setting.power deleted
2020-12-15 10:55:27.812 - info: tado.0 (5657) state tado.0.580325.Rooms.6.overlay.clearZoneOverlay deleted
2020-12-15 10:55:27.823 - info: tado.0 (5657) state tado.0.580325.Rooms.6.overlay.termination.typeSkillBasedApp deleted
2020-12-15 10:55:59.551 - error: tado.0 (5657) Error in data refresh : Error: Request failed with status code 504
2020-12-15 10:55:59.552 - error: tado.0 (5657) Disconnected from Tado cloud service ..., retry in 30 seconds !
2020-12-15 10:56:30.165 - info: tado.0 (5657) Connected to Tado cloud, initialyzing ...
2020-12-15 10:57:30.620 - error: tado.0 (5657) Error in data refresh : Error: Request failed with status code 504
2020-12-15 10:57:30.620 - error: tado.0 (5657) Disconnected from Tado cloud service ..., retry in 30 seconds !
2020-12-15 10:58:01.168 - info: tado.0 (5657) Connected to Tado cloud, initialyzing ...
2020-12-15 10:58:49.522 - info: tado.0 (5657) Initialisation finished, connected to Tado Cloud service refreshing every : 10 seconds
2020-12-15 11:00:00.030 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 11:11:52.621 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Balkon Wohnbereich geöffnet
2020-12-15 11:11:52.622 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer = manual
2020-12-15 11:11:52.622 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer = manual
2020-12-15 11:11:52.623 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer Heizung ausgeschaltet
2020-12-15 11:11:52.624 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer Heizung ausgeschaltet
2020-12-15 11:11:52.640 - info: tado.0 (5657) Power changed for room : 1 in home : 580325 to API with : off and Temperature : 19 and mode : MANUAL
2020-12-15 11:11:52.646 - info: tado.0 (5657) Power changed for room : 2 in home : 580325 to API with : off and Temperature : 18 and mode : MANUAL
2020-12-15 11:21:38.670 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Balkon Wohnbereich geschlossen
2020-12-15 11:21:38.671 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer Heizung auf manual
2020-12-15 11:21:38.672 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer Heizung auf manual
2020-12-15 11:21:38.727 - info: tado.0 (5657) Power changed for room : 1 in home : 580325 to API with : on and Temperature : 19 and mode : MANUAL
2020-12-15 11:21:38.775 - info: tado.0 (5657) Power changed for room : 2 in home : 580325 to API with : on and Temperature : 18 and mode : MANUAL
2020-12-15 12:00:00.034 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 12:40:05.755 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Balkon Wohnbereich geöffnet
2020-12-15 12:40:05.756 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer = manual
2020-12-15 12:40:05.756 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer = manual
2020-12-15 12:40:05.758 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer Heizung ausgeschaltet
2020-12-15 12:40:05.759 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer Heizung ausgeschaltet
2020-12-15 12:40:05.768 - info: tado.0 (5657) Power changed for room : 1 in home : 580325 to API with : off and Temperature : 19 and mode : MANUAL
2020-12-15 12:40:05.776 - info: tado.0 (5657) Power changed for room : 2 in home : 580325 to API with : off and Temperature : 18 and mode : MANUAL
2020-12-15 12:52:07.936 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Balkon Wohnbereich geschlossen
2020-12-15 12:52:07.939 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Esszimmer Heizung auf manual
2020-12-15 12:52:07.940 - info: javascript.0 (5717) script.js.Thermostate.Wohnbereich_Heizung: KEVQU: Wohnzimmer Heizung auf manual
2020-12-15 12:52:07.954 - info: tado.0 (5657) Power changed for room : 1 in home : 580325 to API with : on and Temperature : 19 and mode : MANUAL
2020-12-15 12:52:07.961 - info: tado.0 (5657) Power changed for room : 2 in home : 580325 to API with : on and Temperature : 18 and mode : MANUAL
2020-12-15 13:00:00.030 - info: javascript.0 (5717) script.js.Tablet_Laden: KEVQU: Prüfe Tablet Akkuzustand
2020-12-15 13:15:51.924 - warn: tado.0 (5657) Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false
2020-12-15 13:15:53.448 - warn: tado.0 (5657) Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false
2020-12-15 13:15:54.812 - warn: tado.0 (5657) Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false
2020-12-15 13:15:56.071 - warn: tado.0 (5657) Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false
2020-12-15 13:15:57.499 - warn: tado.0 (5657) Send this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false

**Comment by HGlab01** [comment]
Hi! Please test
https://github.com/HGlab01/ioBroker.tado/archive/0.0.15.tar.gz
(by using the "cat" to install direkt via link; version will be 0.2.4)
If it works I will create a Pull Request.
Thanx!

**Comment by Schmetterfliege** [comment]
@HGlab01 
hope I didn't do anything wrong.
Copied the file to the pi home folder, executed it with cat.
a lot of non-sense shown, but no error. (example see below)
Reloaded the adapter -> still spammed in the log

���5��<6��]���xd����(�6M`N�b�~�˽�]��=z�i��t��H��/�����ֱ4;њ�_O��!�= ����[��$�z(\��&�GY���U�o�'�P�G,m��+��_����"���E��sy)6o��=����f�%���o�I��ǿ�*q�$� �Q�R��;�a��Q�g��Ca&d?��%b�����c懤�1���-��G[�*"��
�:O���n���ekHgސ-�       4��8��2׽G���ߒ�j�<6ɿ7��c��[��l���Kݰ%]���0����e`��[�z��9���Eb����B������4�w��~w��nl�ޔWA�]'eϓyQ~���ď9m�^b�(
����T��Mz����uzD<d$e���� ��|��?���'�}M�N�X*�Ÿ�S"^�҄x&O|K�2wA@^��E������  ��
ʆDF�UO��3K�)�2���7e�-vq$�-�R�'���𳅘�p��9K`

**Comment by smartiebs** [comment]
Hi @HGlab01 
in my case the update works fine and no more warnings are displayed in the log. Thanks for your work!

**Comment by HGlab01** [comment]
@Schmetterfliege 
Very strange! I installed directly from the URL on my test-system and have a normal behavior in the log.
Can you uninstall first and install it afterwards directly from the URL like here:
![image](https://user-images.githubusercontent.com/54232526/102257802-7d0c1280-3f0d-11eb-8eed-47df81c47dce.png)

**Comment by HGlab01** [comment]
> Hi @HGlab01
> in my case the update works fine and no more warnings are displayed in the log. Thanks for your work!

This behaves as expected :-)

**Comment by sly-1** [comment]
It seems to work for me as well. 
After the installation, the message is no longer displayed in the log. 
Thanks for fixing this.

**Comment by Schmetterfliege** [comment]
Uninstalled tado adapter, reinstalled tado adapter -> same log spam.
Uninstalled tado, installed via your github link -> exited with error code 0 (success) -> no tado instance found.
Installed again via your github link -> same "success" but still no tado instance found.
Installed tado adapter (regularly) ->same log spam.
Then installe via your github (without removing the "original" adapter first) -> no log spam anymore :)

**Comment by HGlab01** [comment]
PR #129 created

**Comment by HGlab01** [comment]
> Edit:
> only successful way for me is to update via ioBroker while tado is there.
> Updating with cat (manually) failed. Installing without "original" tado being there failed too.

No worries, about the install issue. As soon as the PR is merged and a new npm-version is available, the standard installation procedure will work again

**Comment by Grothesk242** [comment]
@HGlab01 Your patched code is running fine on my system.

**Comment by DutchmanNL** [comment]
> PR #129 created

thanks merged, I will publish a new version to NPM which will be available today after 4PM

**Comment by senger1985** [comment]
Will be this State writable?

**Comment by DutchmanNL** [comment]
> Will be this State writable?

Not in current way of implementation Not sure if the api supports that at all

@HGlab01 what do you think ?

**Comment by senger1985** [comment]
I think that it blocks the thermostats to manually change the temperature.
This function has long been expected by "tadoists".
For information "it's not published in the app yet, only published in the API"

**Comment by HGlab01** [comment]
> > Will be this State writable?
> 
> Not in current way of implementation Not sure if the api supports that at all
> 
> @HGlab01 what do you think ?

I would not start with this as long as
1.  stable adapter release was launched
2. Tado has implemented this feature in their consumers (app, web) and stabilized it. Tado often do not care about breaking API changes.

Mid-term I guess it is a nice project ;-)
By the way: My Tado even does not provide the new attribute. As of now I am not even sure if put method is supported for this ressource.

**Comment by sly-1** [comment]
I also get this error message: 
`Send  this info to developer !!! { Unhandable information found in DoReadDevices : "childLockEnabled" with value : false`
But even though I have installed the latest version **v0.2.6-1** https://github.com/HGlab01/ioBroker.tado/releases/tag/v0.2.6-1 from the archive. 
The problem should have been fixed in #128. 
This fix does not seem to be included in **v0.2.6-1**. 
Does anyone have any ideas about this?
